src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -493,6 +493,13 @@ public abstract class WebRequestPSCmdlet : PSCmdlet, IDisposable
         [Parameter]
         public virtual SwitchParameter SkipHttpErrorCheck { get; set; }
 
+        /// <summary>
+        /// Gets or sets whether to show the request content in the debug output.
+        /// Only applicable if -Debug is also specified.
+        /// </summary>
+        [Parameter]
+        public virtual SwitchParameter DebugRequestContent { get; set; }
```

Comment:
Since WG has not discussed or approved this parameter this cannot be merged and may take a long time.
As I said in related issue I prefer to start with simplest way (just a list) to get fast feedback from users.

Reply:
As a side note, I think it's more important for debugging to see the content encoding (MIME) header and content size than the content itself.

Reply:
Content is often very important when debugging REST API calls. For example, realizing the hashtable got converted to JSON wrong or is not capitalized.

I'm OK with removing it for now and adding the content part in a future PR.

Reply:
I see [HttpRequestMessage.Tostring()](https://source.dot.net/#System.Net.Http/System/Net/Http/HttpRequestMessage.cs,163) outputs content type. So we could add content size today.